lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -11,6 +11,8 @@ def initialize(input, suspended_action:)
         @playbook = input.values.first[:input][:action_input][:script]
         @root = working_dir
         @verbosity_level = input.values.first[:input][:action_input][:verbosity_level]
+        @check_mode = input.values.first[:input][:action_input][:ansible_inventory][:all][:vars]['check mode']
+        @check_mode = input.values.first[:input][:action_input][:check_mode] if @check_mode.empty?
```

Comment:
If it's not run from form, but the run roles button, this will fail with `<NoMethodError>: undefined method `empty?' for nil:NilClass`
```suggestion
        @check_mode = input.values.first[:input][:action_input][:check_mode] if @check_mode.blank?
```

Reply:
@ezr-ondrej This is very weird, as I know blank? should work on any object but when I changed it to blank? I received this error: <NoMethodError> undefined method `blank?' for "no":String.
Do you maybe have an idea why this is happening?

Reply:
@ezr-ondrej After another look it seems that blank? method works on every object in rails project only.
I used empty? and nil? instead. Let me know if it looks ok for you.

Reply:
huh, right, I didn't realize we are out of rails (the code mixture in one repo is very confusing...)
`@check_mode.nil? || @check_mode.empty?` should be fine I guess unless the input can be a real boolean though